zwave_js/CHANGELOG.md

```diff
@@ -1,5 +1,70 @@
 # Changelog
 
+## 0.1.85
+
+### New features
+
+- Z-Wave JS Server: Add support for inclusions that are started outside of Z-Wave JS Server
+- Z-Wave JS Server: Add support for lastSeen property
+- Z-Wave JS: Improve support for "identify" commands
+- Z-Wave JS: Add API to discover node neighbors
+- Z-Wave JS: Add support for assigning custom return routes and reading previously set priority return routes from cache

```

Comment:
Assuming the answer to the other questions is yes, I consider those most relevant. The network management / routing stuff is not exposed in HA:
```suggestion
### New features

- Z-Wave JS Server: Add support for inclusions that are started outside of Z-Wave JS Server
- Z-Wave JS Server: Add support for lastSeen property
- Z-Wave JS: Add support for identifying on request of other nodes
- Z-Wave JS: Improved auto-discovery of config parameters
```